.github/CODEOWNERS

```diff
@@ -9,3 +9,6 @@
 
 # Beats
 /resources/approval-list/beats.yml @andresrc
+
+# Auto assign reviews
+* @elastic/observablt-robots @elastic/observablt-robots-contractors
```

Comment:
Unrelated to this PR specifically, but FWIW I don't see why we should have two separate groups at all.

Reply:
Exactly :)